source/customization/customizations.rst

```diff
@@ -237,6 +235,71 @@ If you access the Dashboard, and it crashes, then you may have made a mistake
 in ``ood.rb`` file, whose code is run during the initialization of the Rails
 app.
 
+.. _add-menu-items-to-the-navbar:
+
+Add Menu Items to the Navbar
+-----------------------------
+
+  Menus/Sub-menus are viewed as apps in Open Ondemand.  For every menu/sub-menu you want to show in the navbar, you will need to do the following for each item.
+
+  * sudo mkdir /var/www/ood/apps/sys/<NEW MENU ITEM>
+  * sudo vi /var/www/ood/apps/sys/<NEW MENU ITEM>/manifest.yml
+  
+  .. code-block:: yaml
+
+    ---
+    name: Add Menu Items to the navbar   # This will show as the menu item.
+    category: Links  # specify the main menu for the item to be under.  If it doesn't exist, it will be created.
```

Comment:
It won't be created automatically. Maybe a note here or somewhere in this section that you need to refer to the next section to ensure that this category shows in the navigation bar.

Reply:
Actually. It does create them automatically. I had to take no extra steps.

Reply:
<html>
<body>
<!--StartFragment-->

Default configuration | “Files”, “Jobs”, “Clusters”, “Interactive Apps”, “Reports” | whitelist mode is false, so whitelist now only enforces sort order
-- | -- | --


<!--EndFragment-->
</body>
</html>

Reply:
I stand corrected! No I didn't test, I just could of swore that `false` was the default.